src/tests/test_enrich.py

```diff
@@ -5,14 +5,14 @@
 
 from gobupload.storage.handler import GOBStorageHandler
 from gobupload.enrich import enrich, _autoid
+from gobupload import logger
 
-class MockLogger():
-    def info(self, msg):
-        pass
+magic_loggers = MagicMock()
+magic_loggers.get = MagicMock(return_value=None)
+logger.override_loggers(magic_loggers)
```

Comment:
mock get_logger instead of introducing a method in GobLogger that is only for testing